packages/subgraph/schema.graphql

```diff
@@ -220,6 +197,25 @@ type Dao @entity {
   plugins: [PluginInstallation!]! @derivedFrom(field: "dao")
 }
 
+# ACL
+
+# Cannot be immutable because a permission can be revoked and granted in the same TX.
+# This results in an error in the subgraph if the entity is immutable.
+# No need to store granted as we can delete permission when revoked
+type Permission @entity {
+  id: ID! # emitting contract Address (DAO or PluginRepo) + where + permissionId + who
```

Comment:
shouldn't the ID be the same structure as the function parameters:
`emitting_contract + permissionID + where + who `
?

Reply:
amended